AirCasting/Graph/Graph.swift

```diff
@@ -41,6 +41,10 @@ struct Graph: UIViewRepresentable {
     var isAutozoomEnabled: Bool
     let simplifiedGraphEntryThreshold = 1000
     
+    private var properThresholds: [Float] {
+        stream.isTemperature && userSettings.convertToCelsius ? thresholds.rawThresholdsBindingCelsius.wrappedValue : thresholds.rawThresholdsBinding.wrappedValue
```

Comment:
I see this kind of code repeating multiple times across this PR. This is not scalable and very fragile solution. Imagine we'll be tasked to add more settings like that in the future (the simplest example would be "add the possibility to display temperature in Kelvin"). All this code would inflate to a pile of extremely hard to read `if/else` statements real quick and would be really fragile (meaning a mistake would be super easy to make). This is a code smell and it should be packed into a single place somehow. (maybe some `ThresholdFormatter` class that would take a session in and have all the variable stuff like values, colors etc.?)
Please think about something and replace all places you're doing this kind of checks with this better solution